fetch/main.js

```diff
@@ -0,0 +1,26 @@
+async function getData() {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const data = await response.json();
+    console.log(data);
+  } catch (e) {
+    console.error(e);
+  }
+}
+
+async function pokeData() {
+  const response = await fetch('https://pokeapi.co/api/v2/pokemon/1')
+  if (response.ok === 'false'){
```

Comment:
You did it correctly here, except that `response.ok` is a boolean, so this check won't actually work.